docs/proposals/old-timer.md

```diff
@@ -0,0 +1,146 @@
+# Heapster Oldtimer
+
+## Overview
+
+Prior to the Heapster refactor, the Heapster model presented aggregations of
+metrics over certain time periods (the last hour and day).  Post-refactor, the
+concern of presenting an interface for historical metrics was to be split into
+a separate Heapster component: Oldtimer.
```

Comment:
Will this be built into Heapster? Or maybe it'll be running in a separate Pod/Replication Controller/Service?